README.md

```diff
@@ -1 +1,90 @@
-# functional-harness
+## testcafe-functional-harness
+
+Runs functional tests in local browsers and on SauceLabs.
+
+#### What it does
+
+1. Starts the server with the test pages.
+2. Opens the specified browsers (locally or on SauceLabs).
+3. Runs mocha tests.
+4. Closes the browsers and stops the server after all tests are finished.
+
```

Comment:
> If so, why view path - what does it mean?
`Views` is usually used in the context of template frameworks. For example,
there is a template `views/index.ejs' and you render it on the server:
```js
var ejs = require('ejs');
app.get('*', (req, res) => {
    res.render(ejs.render('views/index.ejs'));
});
```

In our case it is a base path to a directory with site pages. (`/tests/page1.html`, '/tests/page2.html'). I suppose it not `baseUrl` but maybe 'basePath' if  you don't like `viewsPath`